### ISSUE TITLE
Pipenv install can't handle spaces in windows user folder

### ISSUE THREAD

**Issue by fluteamahoot** [pypa/pipenv#1618]
Hi there, I've poked my head in on this project to check it out, but I'm having some trouble on my work machine.

##### Describe your environment

1. Windows 10 Pro
1. Python version: Python 3.6.4
1. Pipenv version: pipenv, version 11.1.5
1. Happens in both Powershell and CMD, but not in bash (probably due to Linux Subsystem for Windows uses a different home directory path, that doesn't have spaces in the name)

##### Expected result

pipenv to create the environment

##### Actual result
```
PS D:\Projects\Django\feedback> pipenv --verbose install
Error: no such option: --verbose
PS D:\Projects\Django\feedback> pipenv install --verbose
Creating a Pipfile for this project…
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…

'C:\Users\First' is not recognized as an internal or external command,
operable program or batch file.
```

I've removed my actual name from the output, but to add here, my user folder path is 
"C:\Users\First Last\"

##### Steps to replicate

 1. cd D:\Projects\Django\feedback
 1. pipenv install --verbose

**Comment by techalchemy** [comment]
Possibly cache dir related, what if you run `set PIPENV_CACHE_DIR=D:\Projects\Django\feedback\__cache__` and then `mkdir D:\Projects\Django\feedback\__cache__` and then try?

**Comment by kennethreitz** [comment]
@techalchemy we need to shellquote — i took out your patch because of an error i was getting locally — turns out it wasn't the cause, though.

**Comment by kennethreitz** [comment]
forgot to put it back in

**Comment by fluteamahoot** [comment]
Okay, so setting PIPENV_VENV_IN_PROJECT to true did allow pipenv to install successfully, but now I get
```
PS D:\Projects\Django\feedback> pipenv shell
  File "<string>", line 1
    c:\users\First Last\appdata\local\programs\python\python36-32\python.exe -c "from pew.pew import prevent_path_errors; prevent_path_errors()"

SyntaxError: unexpected character after line continuation character
``` 

Give me a minute and I'll try @techalchemy's suggestion

**Comment by fluteamahoot** [comment]
@techalchemy Here's with the cache moved to the project directory
```
PS D:\Projects\Django\feedback> pipenv install --verbose
Creating a virtualenv for this project…
Using C:\Users\First Last\AppData\Local\Programs\Python\Python36-32\python.exe to create virtualenv…
Running virtualenv with interpreter C:\Users\First Last\AppData\Local\Programs\Python\Python36-32\python.exe
Using base prefix 'C:\\Users\\First Last\\AppData\\Local\\Programs\\Python\\Python36-32'
New python executable in C:\Users\FIRSTL~1\VIRTUA~1\FEEDBA~1\Scripts\python.exe
Installing setuptools, pip, wheel...done.

Virtualenv location: C:\Users\First Last\.virtualenvs\feedback-bBVyDX7t
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…

'C:\Users\First' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by techalchemy** [comment]
@kennethreitz the second bug is the windows shell bug from the last release, we need to get this sorted and cut 11.1.6

**Comment by kennethreitz** [comment]
@techalchemy didn't you fix that already?

**Comment by techalchemy** [comment]
yup, just needs to get cut

**Comment by techalchemy** [comment]
first bug may or may not be fixed by shellquoting

**Comment by techalchemy** [comment]
@fluteamahoot can you `pip install -e git+https://github.com/pypa/pipenv.git@bugfix/windows-bugs#egg=pipenv` and see if this provides a fix

**Comment by W7PEA** [comment]
@techalchemy  I also hit this issue today and your patch fixed it for me (Windows 10, python 3.6.4)  clean install from PIP in down was broken, your patch fixed it.

**Comment by techalchemy** [comment]
@W7PEA thanks for the feedback, awesome! Glad to see we can fix things at least as quickly as we can break them :)
